build_darwin.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+set -eu
+
+BRAVE_TOR_VERSION="1"
+TOR_VERSION="0.3.2.10"
+
+curl -fsSL "https://www.torproject.org/dist/tor-0.3.2.10.tar.gz" -o tor-0.3.2.10.tar.gz
+curl -fsSL "https://www.torproject.org/dist/tor-0.3.2.10.tar.gz.asc" -o tor-0.3.2.10.tar.gz.asc
+
+gpg --keyserver pgp.mit.edu/ --recv 9E92B601
```

Comment:
should also use the long key ID here. see https://github.com/brave/tor_build_scripts/pull/1/files#r187763561